### ISSUE TITLE
Paste event in inputs

### ISSUE THREAD

**Issue by wiuribecomodisimos** [quasarframework/quasar#7191]
**Describe the bug**
I think there's an bug sending the paste event in the inputs, it doesn't trigger, if I add: 
```
    __onPaste (e) {
    ....
      this.$emit('paste', e)
    },

```
 in `quasar/src/components/input/QInput.js`, this works

**Codepen/jsFiddle/Codesandbox (required)**
https://jsfiddle.net/wfpaisa/nw0hr1ck/9/

```
    <q-input
      
      @paste="pasteMethod"
      
    />
```

**To Reproduce**
1. Go to input
2. paste text

**Expected behavior**

**Screenshots**

**Platform (please complete the following information):**
OS:
Node:
NPM:
Yarn:
Browsers:
iOS:
Android:
Electron:

**Additional context**

**Comment by pdanpdan** [comment]
Use `@paste.native` for the moment

**Comment by wiuribecomodisimos** [comment]
Thanks @pdanpdan,  ;)

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.12.6.
